Transformer/ClassNameToNgAdminConfigurationTransformer.php

```diff
@@ -100,6 +101,6 @@ private function getEntityName($className)
         $classParts = explode('\\', $className);
         $entityName = end($classParts);
 
-        return Inflector::tableize($entityName);
+        return Inflector::pluralize(lcfirst($entityName));
```

Comment:
Long term, it might make more sense to look the name up from the registry using the class name.